msvc/mono.def

```diff
@@ -966,3 +966,5 @@ unity_mono_reflection_method_get_method
 mono_unity_g_free
 mono_set_find_plugin_callback
 mono_unity_set_vprintf_func
+mono_type_get_name_full
```

Comment:
I don't recall why these were added. @brianradunity do you know? If not, we can try to revert the changes in this file and see what happens.